apps/docs/src/pages/brand/storybook.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * This page is only used during local development to redirect Storybook links to the Storybook dev server
+ * On production this directory is redirected to the privately hosted Storybook by primer.style
```

Comment:
@pouretrebelle Has this been verified somehow? ☝️. I can't see anything that would prevent this from applying to prod in this diff, but maybe missed it. It applies to previews too, which indicates it would apply to production too.

Reply:
It would apply to prod, but `primer.style/brand/storybook` [redirects to the SB instance](https://github.com/primer/primer.style/blob/main/redirects.json#L887-L896) using a web.config, see https://github.com/primer/primer.style/pull/358. I could put a `if (process.NODE_ENV !== 'development') return null` in there for safety, so it'd show a blank page rather than redirect to localhost and 404 on previews.

Reply:
Gotcha, thanks. The guard seems like a good idea 👍, just so it doesn't inadvertendly affect any other environments. There's a plan to eventually get rid of that reverse proxy in the web.config, so we might get caught out by that eventually.